Fix Typos and Polish

skytin1004 opened the pull request:
Fix Typos and Polish in README, FAQ, MODEL_CARD, CONTRIBUTING, CODE_OF_CONDUCT

vubui commented:
@skytin1004 If you can, could you resolve the conflicts and post an update? Thanks.

skytin1004 commented:
@vubui, @Karliz24
README.md, CODE_OF_CONDUCT.md, CONTRIBUTING.md, UPDATES.md,USE_POLICY.md, MODEL_CARD.md 
I checked and corrected typos in these five markdown files, and replaced some sentences with ones that would make them more readable. While fixing the typos, I also fixed some problems with the markdown grammar. Please check them if you have a chance. Thank you very much.
